be/src/pipeline/exec/partitioned_aggregation_sink_operator.cpp

```diff
@@ -168,13 +168,21 @@ Status PartitionedAggSinkOperatorX::sink(doris::RuntimeState* state, vectorized:
     auto* runtime_state = local_state._runtime_state.get();
     RETURN_IF_ERROR(_agg_sink_operator->sink(runtime_state, in_block, false));
     if (eos) {
-        LOG(INFO) << "agg node " << id() << " sink eos";
-        if (revocable_mem_size(state) > 0) {
-            RETURN_IF_ERROR(revoke_memory(state));
-        } else {
-            for (auto& partition : local_state._shared_state->spill_partitions) {
-                RETURN_IF_ERROR(partition->finish_current_spilling(eos));
+        local_state.profile()->add_info_string(
+                "Spilled", local_state._shared_state->is_spilled ? "true" : "false");
+        LOG(INFO) << "agg node " << id()
```

Comment:
This is debug log?

Reply:
If profile is not enabled, we can seach log to check if query is spilled.